astropy/visualization/units.py

```diff
@@ -100,3 +99,33 @@ def __exit__(self, type, value, tb):
                 units.registry[u.Quantity] = self._original_converter[u.Quantity]
 
     return MplQuantityConverter()
+
+
+# Add the unit_support function to the module's __all__ list
+
+
+def unit_support(format="latex_inline"):
+    """
+    Enable both quantity and time support for matplotlib.
+
+    This is a convenience function that enables both astropy's
```

Comment:
I don't recall seeing that style in our docs, let's avoid it
```suggestion
    This is a convenience function that enables both
```